### PR TITLE
Change GUI colours to suit birthday planners

### LINKED ISSUE #100: 💡 Change color scheme of the application
### Describe the issue
*What is the problem, if any? What is the desired behavior?*

The dark theme is by default. Perhaps change the color theme to something else to personalize our application.

________________________________________________________________
### Additional context
*Describe possible and/or alternative solutions, to highlight required scope of work.*

As simple as modifying `darkTheme.css` under Resources to change the colors.

### PR DESCRIPTION
**Pull request by zhengruoxin**
Modified dark theme to a light pastel theme for a more "birthday party" look. Also changed app icon from contacts book to a party hat. Closes #100.

Note: Draft. dark grey portions of unfilled lists not touched yet. Suggestions welcome! (with the hex code or rgb combi if possible :))
Update: dark grey portions solved, ready for review

Screenshot 2021-03-23 at 11 03 25

### CODE REVIEW

**Review by garyljj — COMMENTED** [review]
Looks good!
Maybe make the new theme a separate file and keep darktheme (for a possible future enhancement to let user switch between themes

**Comment by zhengruoxin** [comment]
> Maybe make the new theme a separate file and keep darktheme (for a possible future enhancement to let user switch between themes

Oyea good idea! Have adopted in latest commit

File: src/main/resources/view/PinkPastelTheme.css [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+    -fx-background-color: derive(#1d1d1d, 20%);
+}
+
+.list-view {
+    -fx-background-insets: 0;
+    -fx-padding: 0;
+    -fx-background-color: derive(#1d1d1d, 20%);
+    -fx-border-color: white;
+    -fx-border-width: 1;
+}
```
Comment (garyljj) [inline src/main/resources/view/PinkPastelTheme.css:96]: Might have missed out this? Not sure if it affects other stuff, but after i changed it looked better haha

```suggestion
.list-view {
    -fx-background-insets: 0;
    -fx-padding: 0;
    -fx-background-color: derive(rgb(243, 229, 231), -10%);
    -fx-border-color: white;
    -fx-border-width: 1;
}
```

**Review by garyljj — COMMENTED** [review]
Maybe for the input field, done use white words? can be a little hard to see
Otherwise looks good

**Review by Ellevy — APPROVED** [review]
LGTM

**Pull request merged**